db/migrate/20170730154117_create_account_migrations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountMigrations < ActiveRecord::Migration[5.1]
+  def change
+    create_table :account_migrations do |t|
+      t.integer :old_person_id, null: false
+      t.integer :new_person_id, null: false
+    end
+
+    add_foreign_key :account_migrations, :people, column: :old_person_id
+    add_foreign_key :account_migrations, :people, column: :new_person_id
+
+    add_index :account_migrations, %i[old_person_id new_person_id], unique: true
```

Comment:
There should also be an index for only `old_person_id`, I don't know if we want to support merging multiple accounts. Currently that wouldn't work (needs more work, merging contacts for example), and is out of scope for now, but that's something that's maybe possible later when doing the import. If not we need a unique index for `new_person_id`, if yes this combined index is enough, because every combination needs to be unique.